front/src/components/spellbookContainer/SpellbookContainer.tsx

```diff
@@ -94,11 +103,24 @@ function SpellbookContainer({
     const spellSummariesLoaded =
         spellSummariesLoadedFromNetwork || spellSummariesLoadedFromLocalStorage;
 
+    function readyDrawerRef(drawerRef: React.RefObject<HTMLDivElement>) {
+        if (drawerRef.current) {
+            drawerRef.current.scrollTop = 0;
+        }
+    }
+
+    function handleOpenDrawer(newDrawerState: DrawerState) {
+        if (newDrawerState === DrawerState.Page) {
+            readyDrawerRef(pageDrawerRef);
+        }
+        onSetDrawerState(newDrawerState);
+    }
```

Comment:
I don't think we need to break this out into two functions.
```suggestion
    function handleOpenDrawer(newDrawerState: DrawerState) {
        if (newDrawerState === DrawerState.Page && pageDrawerRef.current) {
            pageDrawerRef.current.scrollTop = 0;
        }
        onSetDrawerState(newDrawerState);
    }
```